src/js/edu-benefits/1995/config/form.js

```diff
@@ -77,7 +78,6 @@ const formConfig = {
           path: 'benefits-eligibility/benefits-selection',
           initialData: {},
           uiSchema: {
-            'ui:title': 'Benefit selection',
```

Comment:
Just noticed the path is `benefits-eligibility/benefits-selection`, but the title of is `Benefit Selection`. Should the path not have the `s` after `benefit`?